sdks/python/apache_beam/io/parquetio.py

```diff
@@ -83,6 +86,67 @@ def process(self, table, with_filename=False):
         yield row
 
 
+class _RowDictionariesToArrowTable(DoFn):
```

Comment:
I used `pa.Table` for consistency with reading batched parquets, but I wanted to bring up that `pa.RecordBatch` would also work.

I don't have strong preferences either way. I would use it both ways.